src/pipeline_readers.py

```diff
@@ -62,14 +62,14 @@ def read_dlt_delta(spark, bronze_dataflow_spec) -> DataFrame:
         if reader_config_options and len(reader_config_options) > 0:
             return (
                 spark.readStream.options(**reader_config_options).table(
-                    f"""{bronze_dataflow_spec.sourceDetails["database"]}
+                    f"""{bronze_dataflow_spec.sourceDetails["source_database"]}
```

Comment:
Can we introduce a variable, say `source_table` for the value and use it in these two places (so it's clear(er) that they're the same)? 🙏